src/api/sip.ts

```diff
@@ -1,16 +1,16 @@
 import 'brekekejs/lib/jsonrpc'
 import 'brekekejs/lib/webrtcclient'
 
+import { CallOptions, Sip } from 'api/brekekejs'
+import getFrontCameraSourceId from 'api/getFrontCameraSourceId'
+import pbx from 'api/pbx'
+import turnConfig from 'api/turnConfig'
 import EventEmitter from 'eventemitter3'
 import { Platform } from 'react-native'
+import { cancelRecentPn } from 'stores/cancelRecentPn'
+import { BackgroundTimer } from 'utils/BackgroundTimer'
 
 import appPackageJson from '../../package.json'
```

Comment:
This should be handled from env or configuration files, not from the package.json